### ISSUE TITLE
Carrying "et al" from anystyle to Zotero

### ISSUE THREAD

**Issue by md9692** [inukshuk/anystyle#151]
Hi 

Firstly thanks for so swiftly resolving the last issue I raised. I'm not even sure if this is something you can do, or if the fault lies with my bibliography, but:

I have a bibliography, in which any citation with >6 authors is written as the first three, then "et al". 

Anystyle picks up the "et al" (see attachment 1) but then when I create the BibTeX and import that to Zotero, Zotero loses the "et al" information and instead just lists three authors (attachment 2), and then creates references for me in MS Word that also lack the "et al" (attachment 3). 
<img width="1191" alt="1" src="https://user-images.githubusercontent.com/57666469/90966551-732c5280-e4cb-11ea-92ad-6951d76605cd.png">
<img width="330" alt="2" src="https://user-images.githubusercontent.com/57666469/90966554-758eac80-e4cb-11ea-83eb-45e26e923700.png">
<img width="695" alt="3" src="https://user-images.githubusercontent.com/57666469/90966555-76274300-e4cb-11ea-8a4c-a39ac85b361e.png">

Is there anything that can be done about this? Or is the fault with my original bibliography for using "et al"?

**Comment by inukshuk** [comment]
Yes, this is a long-standing nuisance. CSL styles (which are used by Zotero to format references) assume that they have control over the 'et al' in a reference. They obviously need this, otherwise you could not convert, e.g. from a style that uses 'et al' after three names to one that uses it only after four.

There might be a way to override this and force a style to always set 'et al': if that were the case we could explore how to make this flag survive all the way to Zotero. If there's no way to do this, the only option is to try to look up the remaining authors (this is generally the best approach if you are parsing references which already exist in some database).
